Content.Server/Sound/EmitSoundSystem.cs

```diff
@@ -52,4 +73,19 @@ private void HandleEmitSoundOnTrigger(EntityUid uid, EmitSoundOnTriggerComponent
         TryEmitSound(uid, component);
         args.Handled = true;
     }
+
+    private void OnIntervalInit(EntityUid uid, EmitSoundIntervalComponent component, ComponentInit args)
+    {
+        SelectNextInterval(uid, component, true);
+    }
+
```

Comment:
This bulldozes any saved data whenever the game is reloaded so you should use mapinit.

Reply:
Kinda thought so. I'll fix that up.